action/protocol/vote/candidatesutil/candidatesutil.go

```diff
@@ -56,6 +56,9 @@ func CandidatesByHeight(sr protocol.StateReader, height uint64) ([]*state.Candid
 // KickoutListByEpoch returns array of unqualified delegate address in delegate pool for the given epochNum
 func KickoutListByEpoch(sr protocol.StateReader, epochNum uint64) (*vote.Blacklist, error) {
 	blackList := &vote.Blacklist{}
+	if epochNum == 1 {
```

Comment:
@koseoyoung this is a workaround. We can have a better solution on the db level.